kotlinKoans/src/main/java/com/epam/kotlinkoans/i_introduction/_10_Object_Expressions/n10ObjectExpressions.kt

```diff
@@ -18,6 +19,17 @@ fun todoTask10(): Nothing = TODO(
 
 fun task10(): List<Int> {
     val arrayList = arrayListOf(1, 5, 2)
-    Collections.sort(arrayList, todoTask10())
+    Collections.sort(arrayList, object : Comparator<Int>{
```

Comment:
can be simplified 

Collections.sort(arrayList, object : Comparator<Int> {
        override fun compare(o1: Int, o2: Int) = o2.compareTo(o1)
    })